src/init.cpp

```diff
@@ -545,14 +545,14 @@ static void BlockNotifyCallback(bool initialSync, const CBlockIndex *pBlockIndex
 }
 
 static bool fHaveGenesis = false;
-static boost::mutex cs_GenesisWait;
+static CWaitableCriticalSection cs_GenesisWait;
```

Comment:
Could s/CWaitableCriticalSection/Mutex

Reply:
Hmm, that feels weird? We now have CCriticalSection and Mutex which mean two different things, but its not clear /what/ they mean without looking at what they're typedef'd to? Can we just do this as a scripted diff in a separate commit?

Reply:
> Hmm, that feels weird? We now have CCriticalSection and Mutex which mean two different things, but its not clear /what/ they mean without looking at what they're typedef'd to? Can we just do this as a scripted diff in a separate commit?

Scripted-diff renaming followup is at: https://github.com/bitcoin/bitcoin/pull/11599